test/helpers.js

```diff
@@ -254,14 +254,18 @@ describe('helpers', function () {
     })
 
     describe('parameters', function () {
-      it('should not overwrite parameters from path when exists in operation', function () {
+      it('should add parameters from path to operation without overwriting', function () {
```

Comment:
Yup. They are merged. The only thing is if they exist in both (based on both the `name` and the `in` as the unique identifier), then the operation level parameter will override the path level one.